src/main/java/com/minecolonies/coremod/commands/AbstractSingleCommand.java

```diff
@@ -115,7 +115,7 @@ public boolean isPlayerOpped(@NotNull final ICommandSender sender, String cmdNam
      * @param theCommand which command to check if the player can use it
      * @return boolean
      */
-    public boolean canCommandSenderUseCommand(Commands theCommand)
+    public boolean canCommandSenderUseCommand(final Commands theCommand)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of returns of this method 13, down to the maximum allowed 6. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1142)